js/reservation.js

```diff
@@ -0,0 +1,32 @@
+import { validateInput } from "./utils/validation/validation.js";
+import { reservationForm, formInputs } from "./utils/formElements.js";
+import handlePeopleCountChange from "./utils/peopleCount.js";
+
+document.addEventListener("DOMContentLoaded", function () {
+  handlePeopleCountChange();
+  reservationForm.addEventListener("submit", (e) => {
+    e.preventDefault();
+
+    let isValid = true;
+
+    const inputs = [
+      validateInput(formInputs.usernameElem, "no name"),
+      validateInput(formInputs.emailElem, "no email", "email"),
+      validateInput(
+        formInputs.reservationDateElem,
+        "This field is incomplete",
+        "date"
+      ),
+      validateInput(
+        formInputs.reservationTimeElem,
+        "This field is incomplete",
+        "time"
+      ),
+      formInputs.peopleCountElem.innerText,
+    ];
+
+    if (inputs.includes(null)) isValid = false;
+
+    isValid && e.target.submit();
```

Comment:
You can combine both lines and remove the variable isValid like so
```suggestion
   !inputs.includes(null) && e.target.submit();
```

Reply:
Don't forget to remove the variable isValid in line 10

Reply:
I changed it and removed the variable